.github/workflows/build-and-deploy-to-app-engine.viewer.yml

```diff
@@ -1,9 +1,8 @@
 name: Viewer - Build and Deploy to GCP App Engine
 
 on:
-  push:
-    branches:
-      - master
+  release:
+    types: [released]
```

Comment:
_personally_ I would prefer if this is kept to on commit to Master (or develop). Github Releases are just another chore to do if this is set.